test/library/randomized_benchmarking/test_clifford_utils.py

```diff
@@ -916,127 +106,92 @@ def test_number_to_clifford_mapping_single_gate(self):
+            clifford_from_num = CliffordUtils.clifford_2_qubit(compose_2q(num1, num2))
+            clifford_from_circuit = Clifford(cliff1.to_circuit().compose(cliff2.to_circuit()))
+            self.assertEqual(clifford_expected, clifford_from_num)
+            self.assertEqual(clifford_expected, clifford_from_circuit)
 
     def test_inverse_by_num_1q(self):
         """Compare inverse using num to inverse using Clifford"""
-        num_tests = 10
-        rng = default_rng(seed=self.seed)
-        cliff_utils = CliffordUtils(num_qubits=1, basis_gates=self.basis_gates)
-        for _ in range(num_tests):
-            num = rng.integers(0, cliff_utils.NUM_CLIFFORD_1_QUBIT - 1)
-            cliff_qc = cliff_utils.clifford_1_qubit_circuit(num)
-            inverse_num = cliff_utils.clifford_inverse_by_num(num)
-            inverse_by_num = cliff_utils.clifford_1_qubit_circuit(inverse_num)
-            inverse_cliff = Clifford(cliff_qc).adjoint()
-            assert (Operator(inverse_by_num)).equiv(Operator(inverse_cliff))
+        num_tests = 24
+        for num in range(num_tests):
+            cliff = CliffordUtils.clifford_1_qubit(num)
+            clifford_expected = cliff.adjoint()
+            clifford_from_num = CliffordUtils.clifford_1_qubit(inverse_1q(num))
+            clifford_from_circuit = Clifford(cliff.to_circuit().inverse())
+            self.assertEqual(clifford_expected, clifford_from_num)
+            self.assertEqual(clifford_expected, clifford_from_circuit)
 
     def test_inverse_by_num_2q(self):
         """Compare inverse using num to inverse using Clifford"""
         num_tests = 100
         rng = default_rng(seed=self.seed)
-        cliff_utils = CliffordUtils(num_qubits=2, basis_gates=self.basis_gates)
         for _ in range(num_tests):
-            num = rng.integers(0, cliff_utils.NUM_CLIFFORD_2_QUBIT - 1)
-            cliff_qc = cliff_utils.clifford_2_qubit_circuit(num)
-            inverse_num = cliff_utils.clifford_inverse_by_num(num)
-            inverse_by_num = cliff_utils.clifford_2_qubit_circuit(inverse_num)
-            inverse_cliff = Clifford(cliff_qc).adjoint()
-            assert (Operator(inverse_by_num)).equiv(Operator(inverse_cliff))
-
-    def is_permutation(self, num_elements, perm):
-        """Check that the input list is a permutation of the numbers [0, num_elements]."""
-        all_nums = {i: False for i in range(num_elements)}
-        for n in all_nums:
-            if all_nums[perm[n]]:
-                raise QiskitError("Not a permutation")
-
-            all_nums[perm[n]] = True
-        assert all(x is True for x in all_nums.values())
-
-    def test_layers_to_num(self):
-        """Check that all 2 clifford numbers form a permutation over [0, 11519]"""
-        cliff_utils = CliffordUtils(num_qubits=2, basis_gates=self.basis_gates)
-        cliff_utils.transpile_2q_cliff_layers()
-        self.is_permutation(cliff_utils.NUM_CLIFFORD_2_QUBIT, CLIFF_LAYERS_TO_NUM_2Q)
```

Comment:
Testing that rows for a permutation was requested by @ShellyGarion in her initial review of my code.

Reply:
As far as I understand, this test was necessary to check the validity of `CLIFF_LAYERS_TO_NUM_2Q` that defines another mapping between number and Clifford than a mapping defined in `CliffordUtils.clifford_2_qubit`.

I've removed `CLIFF_LAYERS_TO_NUM_2Q` and changed to have only one mapping between number and Clifford, so I think we don't need the test any longer ([this round-trip test](https://github.com/Qiskit/qiskit-experiments/blob/a9c1760f0b9a0fc3426c2337a2a9c624696ca221/test/library/randomized_benchmarking/test_clifford_utils.py#L116) would be sufficient to check the validity of the mapping).
I'll let users know the change in number-2qClifford mapping via release note.

Reply:
I am not sure I fully understand the new mapping, so maybe I am mistaken. For 1 qubit, we expect the mapping of all Cliffords to contain all the numbers 0-23 in some permutation. Is this still correct?

Reply:
Yes. It must be correct. 
For 1Q case, `test_clifford_1_qubit_generation` directly guarantees that [0..23] yields all possible 24 Cliffords.
For 2Q case, `test_number_to_clifford_mapping_2q` effectively guarantees that [0..11519] yields all possible 11520 Cliffords. Instead of enumerating all symplectic stabilizer tables, I used the round trip test (number i -> Clifford -> number i). Its success suggests all possible 11520 Cliffords are generated.